CONTRIBUTING.md

```diff
@@ -32,8 +32,8 @@ Before we get started, here are a few things we expect from you (and that you sh
 
 * Be kind and thoughtful in your conversations around this project. We all come from different backgrounds and projects, which means we likely have different perspectives on "how open source is done." Try to listen to others rather than convince them that your way is correct.
 * Open Source Guides are released with a [Contributor Code of Conduct](./CODE_OF_CONDUCT.md). By participating in this project, you agree to abide by its terms.
-* If you open a pull request, please ensure that your contribution passes all tests. If there are test failures, you will need to address them before we can merge your contribution.
-* When adding content, please consider if it is widely valuable. Please don't add references or links to things you or your employer have created as others will do so if they appreciate it.
+* Please ensure that your contribution passes all tests if you open a pull request. If there are test failures, you will need to address them before merging your contribution.
+* When adding content, please consider if it is widely valuable. Please don't add references or links to things you or your employer have created, as others will do if they appreciate it.
```

Comment:
```suggestion
* Please ensure that your contribution passes all tests if you open a pull request. If there are test failures, you will need to address them before we can merge your contribution.
* When adding content, please consider if it is widely valuable. Please don't add references or links to things you or your employer have created, as others will do so if they appreciate it.
```

I'm on board with most of the rephrasing but:

* Contributors don't have permission to merge their own contributions, so changing "we can merge" to "merging" doesn't make sense; and
* I think the second line still reads better with the "so" in place (but adding the comma is 👍🏻 )

Reply:
All right that's OK.